src/GraphicsControls/Handlers/CheckBox/CheckBoxHandler.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Linq;
+
+namespace Microsoft.Maui.Graphics.Controls
+{
+	public class CheckBoxHandler : GraphicsControlHandler<ICheckBoxDrawable, ICheckBox>
+	{
+		public static PropertyMapper<ICheckBox> PropertyMapper = new PropertyMapper<ICheckBox>(ViewHandler.Mapper)
+		{
+			Actions =
+			{
+				[nameof(ICheckBox.IsChecked)] = ViewHandler.MapInvalidate
+			}
+		};
+
+		public static DrawMapper<ICheckBoxDrawable, ICheckBox> DrawMapper = new DrawMapper<ICheckBoxDrawable, ICheckBox>(ViewHandler.DrawMapper)
+		{
+			["Background"] = MapDrawBackground,
+			["Mark"] = MapDrawMark,
+			["Text"] = MapDrawText
+		};
+
+		public CheckBoxHandler() : base(DrawMapper, PropertyMapper)
+		{
+
+		}
+
+		public static string[] DefaultCheckBoxLayerDrawingOrder =
+			ViewHandler.DefaultLayerDrawingOrder.ToList().InsertAfter(new string[]
+			{
+				"Background",
```

Comment:
Not that this is particularly related to this PR's purpose, but when I see the same string literal in a file multiple times, I worry about accidental typos getting them out of sync (a very subtle bug to find). I wonder if, _since this is an example of how to do things in MAUI_, we should be using `const string` inside this class for these property names?

Maybe we should even expose theme as properties on `ICheckBoxDrawable` and use the `[nameof]ICheckboxDrawable.Background` pattern we see in line 11?